api/src/services/es-service.ts

```diff
@@ -23,7 +23,7 @@ export class ESService {
    */
   async getEsClient(): Promise<Client> {
     if (!this.esClient) {
-      this.esClient = await new Client({ node: process.env.ELASTICSEARCH_URL });
+      this.esClient = await new Client({ node: process.env.BACKBONE_ELASTICSEARCH_URL });
```

Comment:
If you expect `BACKBONE_ELASTIC_URL` to be defined in dev/test/prod, make sure you add it to the appropriate files. See https://github.com/bcgov/biohubbc-platform/pull/96